packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1957,6 +1967,8 @@ export default class Meeting extends StatelessWebexPlugin {
     });
     this.locusInfo.on(LOCUSINFO.EVENTS.SELF_UNADMITTED_GUEST, (payload) => {
       if (payload) {
+        this.startKeepAlive();
```

Comment:
Does this only apply for guest or also for a users who are put in lobby and waiting for host to join !?

Reply:
can you confirm for the guest users who are in lobby but does not need to be admitted , This wont get triggred

Reply:
This is triggered for a space member put in the lobby of a locked meeting waiting to be admitted.
It is also triggered for users who are in a lobby and automatically admitted when the host enters the meeting.

Reply:
thanks for confirming